### ISSUE TITLE
Document changes from upstream 

### ISSUE THREAD

**Issue by sevku** [labthings/picamerax#9]
You write in your introduction that awb greyworld is supported, but I can not find any details about it in the documentation.

**Comment by sevku** [comment]
ok I found the relevant pull request and figured it out:

`camera._set_awb_mode('greyworld')`

Would be nice if this could be added to the documentation.

**Comment by mysteriousHerb** [comment]
indeed. the documentation seems to be identical to the original picamera library. It would be nice if there is a separate page just to talk about the usage of added functions..

**Comment by jtc42** [comment]
Thanks, yeah I need to document this really. Unfortunately I’m no longer on the project this fork was made for, so I have limited time available on this.

What I’ll do is try and add docs for the changes to the readme at the very least.
